lib/vector-buffers/proptest-regressions/variants/disk_v2/tests/model/mod.txt

```diff
@@ -10,3 +10,6 @@ cc 80077436fe364525dcab4e5878d617db14fae6e23a7e79f820ed592ce4614a8d # shrinks to
 cc 4b521a9a467f44e9cbc30d830624c03605522f0310e3e972409ac7f0c42d20cd # shrinks to config = DiskBufferConfig { data_dir: "/tmp/vector-disk-v2-model", max_buffer_size: 64, max_data_file_size: 2, max_record_size: 1714, write_buffer_size: 61440, flush_interval: 10s, filesystem: TestFilesystem { inner: Mutex { data: FilesystemInner { files: {} } } } }, actions = [WriteRecord(Record { id: 0, size: 5, event_count: 1 }), WriteRecord(Record { id: 0, size: 0, event_count: 1 }), WriteRecord(Record { id: 0, size: 0, event_count: 1 })]
 cc 99f23327ef7e759a9cb6424a3fa3495aaf4f1c7ef23145b5f5d72eb6cc5e0173 # shrinks to config = DiskBufferConfig { data_dir: "/tmp/vector-disk-v2-model", max_buffer_size: 64, max_data_file_size: 2, max_record_size: 61398, write_buffer_size: 61440, flush_interval: 10s, filesystem: TestFilesystem { inner: Mutex { data: FilesystemInner { files: {} } } } }, actions = [WriteRecord(Record { id: 0, size: 61381, event_count: 1 })]
+cc 54be1ee096dc5169013a1cd8a114f23c5aca7a209a663d783c7f74b4c4ff4746 # shrinks to mut config = DiskBufferConfig { data_dir: "/tmp/vector-disk-v2-model", max_buffer_size: 153516, max_data_file_size: 14228, max_record_size: 14214, write_buffer_size: 61440, flush_interval: 10s, filesystem: TestFilesystem { inner: Mutex { data: FilesystemInner { files: {} } } } }, actions = [WriteRecord(Record { id: 0, size: 14163, event_count: 1 })]
+cc d812fdee8da4aae904579f08cadae1b585944da58040cad4f18544a87faf240e # shrinks to mut config = DiskBufferConfig { data_dir: "/tmp/vector-disk-v2-model", max_buffer_size: 53736, max_data_file_size: 46808, max_record_size: 28912, write_buffer_size: 61440, flush_interval: 10s, filesystem: TestFilesystem { files: {} } }, actions = [WriteRecord(Record { id: 0, size: 23565, event_count: 1, aligned_len: 23632 }), WriteRecord(Record { id: 0, size: 18445, event_count: 1, aligned_len: 18512 }), WriteRecord(Record { id: 0, size: 11557, event_count: 1, aligned_len: 11616 })]
+cc b08fb47ac81e9148d8dc6a4d3332e92c21751685be9da9bd0c5d962ad7436285 # shrinks to mut config = DiskBufferConfig { data_dir: "/tmp/vector-disk-v2-model", max_buffer_size: 69602, max_data_file_size: 2462, max_record_size: 2450, write_buffer_size: 61440, flush_interval: 10s, filesystem: TestFilesystem { files: {} } }, actions = [WriteRecord(Record { id: 0, size: 2389, event_count: 1, archived_len: 2464 })]
```

Comment:
This might be might favorite thing about proptest, the regression files.

Reply:
Big agree.